Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -62,6 +63,11 @@ def pr_pull_args
       flag   "--bintray-mirror=",
              description: "Use the specified Bintray repository to automatically mirror stable URLs "\
                           "defined in the formulae (default: `mirror`)."
+      comma_array "--workflows=",
```

Comment:
Does 

```suggestion
      comma_array "--workflow=", "--workflows=",
```

work?

Reply:
Wrong number of arguments. comma_array expects only 1 argument.